package.json

```diff
@@ -8,25 +8,23 @@
   },
   "files": [
     "lib",
-    "lib-es6",
     "react-mf.d.ts"
   ],
-  "main": "lib/index.js",
+  "main": "lib/index.cjs.js",
   "keywords": [
     "react",
     "molecular formula",
     "mf renderer"
   ],
-  "module": "lib-es6/index.js",
+  "module": "lib/index.es.js",
```

Comment:
Is it possible to remove the `.js` suffix?

Reply:
I mean in the actual filename, not just here.

Reply:
If possible, then we should have `index.cjs` and `index.mjs`.

Reply:
I did not found a solution to do it with one run. So I took the same approach you had with babel :
Two build with an env variable to tweak the build config.